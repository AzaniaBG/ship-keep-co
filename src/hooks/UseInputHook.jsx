import { useState } from 'react';

export default function() {

    return (
        <input
            data-testid="input"
            type="text"
            placeholder="name"
        />
    );
}
