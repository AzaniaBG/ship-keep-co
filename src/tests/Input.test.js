import renderer from 'react-test-renderer';
import UseInputHook from '../hooks/UseInputHook';

it('changes based on user input', () => {
    const component = renderer.create(
        <UseInputHook />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
