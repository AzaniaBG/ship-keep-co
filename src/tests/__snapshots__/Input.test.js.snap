// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes based on user input 1`] = `
<input
  data-testid="input"
  placeholder="name"
  type="text"
/>
`;
